### PR TITLE
Remove test-app-insights-logger and load-test collateral

### PR DESCRIPTION
**Pull request by ChumpChief**
~~Opening as a draft, checking with several stakeholders to make sure this isn't being used in an unexpected way.~~ Checked with @GaryWilber who confirmed this is no longer used.

It appears that the test-app-insights-logger package is only referenced from the stress tests, but from a Dockerfile which itself appears unreferenced.  #8704 removed what appears to have been the only reference to it about 2.5 years ago.

Similarly, with that deployLoadTestToAks.yml file removed, I suspect the other files here are also no longer used (i.e. I suspect runLoadTestOnAks.ps1 expected deployLoadTestToAks.yml to have run).

This PR would remove that load-test collateral, as well as the test-app-insights-logger package as it would become unreferenced.

[AB#11800](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/11800)

### CODE REVIEW

File: fluidBuild.config.cjs [hunk 1]
```diff
@@ -440,7 +439,6 @@ module.exports = {
 				// internal partners or internal CI requirements.
 				internalFeed: [
 					// TODO: We may not need to publish test packages to the internal feed, remove these exceptions if possible.
```
Comment (markfields) [inline fluidBuild.config.cjs:441]: Remove this TODO?

Comment (ChumpChief) [reply fluidBuild.config.cjs:441]: It's still relevant for `@fluid-internal/test-service-load` which remains.

**Review by markfields — APPROVED** [review]
LGTM

File: PACKAGES.md
Comment (alexvy86) [inline PACKAGES.md:1]: What's with all the changes here? They don't seem related...

**Review by alexvy86 — APPROVED** [review]
LGTM except the PACKAGES.md changes which I don't understand.

File: PACKAGES.md [hunk 2]
```diff
@@ -27,13 +27,13 @@ The dependencies between layers are enforced by the layer-check command._
...
 | --- | --- |
-| - [@fluidframework/driver-definitions](/packages/common/driver-definitions)</br>- [@fluidframework/odsp-driver-definitions](/packages/drivers/odsp-driver-definitions) | - [Core-Interfaces](#Core-Interfaces)</br>- [Protocol-Definitions](#Protocol-Definitions) |
+| - [@fluidframework/driver-definitions](/packages/common/driver-definitions)</br>- [@fluidframework/odsp-driver-definitions](/packages/drivers/odsp-driver-definitions) | - [Core-Interfaces](#Core-Interfaces)</br>&nbsp; |
 
 ### Container-Definitions
 
 | Packages | Layer Dependencies |
 | --- | --- |
-| - [@fluidframework/container-definitions](/packages/common/container-definitions)</br>&nbsp;</br>&nbsp; | - [Core-Interfaces](#Core-Interfaces)</br>- [Protocol-Definitions](#Protocol-Definitions)</br>- [Driver-Definitions](#Driver-Definitions) |
+| - [@fluidframework/container-definitions](/packages/common/container-definitions)</br>&nbsp; | - [Core-Interfaces](#Core-Interfaces)</br>- [Driver-Definitions](#Driver-Definitions) |
```
Comment (ChumpChief) [inline PACKAGES.md:36]: @alexvy86 PACKAGES.md should be regenerated (`pnpm layer-check --md .`) after any addition/removal of a package, or any change to layerInfo.json.  Most engineers forget to do this, so this includes many unrelated updates from those omissions.  For random example, this line change originates from this PR here: https://github.com/microsoft/FluidFramework/pull/21200/files#diff-33f48800ac63780832d79c24ea32441d5b470c10b6cf9871116bac70a3925e7aR69

File: PACKAGES.md
Comment (ChumpChief) [reply PACKAGES.md:1]: I need to regenerate PACKAGES.md since I'm removing a package, and this is picking up all the other changes since the last time it was regenerated (others should have regenerated but didn't).
